display.py

```diff
@@ -145,11 +158,32 @@ def runGame(self, numbots):
 					DrawButton.disable()
 					my_player.stay(gamestate)
 					gamestate.waitforplayer = False
+				if ResolveButton.check(event):
+					# end game when all players done
+					if all([not player.playing for player in gamestate.players]):
+						self.showPostgame()
```

Comment:
You probably want to return from here also so we break out of this function completely and back to the main menu.

Reply:
Thank you I haddn't quite wrapped my head around the loops within loops. I added the return and added a STATE_POST for the postgame and it seems to work now